App.js

```diff
@@ -30,10 +32,74 @@ import {
 import { Colors } from "react-native/Libraries/NewAppScreen";
 
 function HomeScreen() {
+  const [length, setLength] = useState(0);
+  const [width, setWidth] = useState(0);
+  const [gram, setGram] = useState(0);
+  const [currentCount, setCurrentCount] = useState(0);
+  const [weight, setWeight] = useState(0);
+
+  const [selected, setSelected] = useState("DIN A");
+  const [selectedSecondary, setSelectedSecondary] = useState(false);
+
+  const handleOnSelection = (button) => {
+    setSelected(button.name);
+    setSelectedSecondary(false);
+  };
+
+  const weightHandler = () => {
+    setWeight(length * width * 0.01 * 0.01 * gram * currentCount);
+  };
+
+  const handleOnSelectionSecondary = (title) => {
+    setSelectedSecondary(title);
+    if ([selectedSecondary] in Size[selected.replace(/\s/g, "")]) {
+      setLength(Size[selected.replace(/\s/g, "")][selectedSecondary]["length"]);
+      setWidth(Size[selected.replace(/\s/g, "")][selectedSecondary]["width"]);
+      setGram(Size[selected.replace(/\s/g, "")][selectedSecondary]["gram"]);
+      setWeight(parseInt(length * width * 0.01 * 0.01 * gram * currentCount));
+    }
+  };
+
+  const handleOnChangeLength = (val) => {
```

Comment:
Please use this convention and format everywhere.
**`handleActionPerformed`**
```suggestion
  const handleOnLengthChanged = (val) => {
```